vagrant/Vagrantfile

```diff
@@ -0,0 +1,11 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "debian/contrib-jessie64"
```

Comment:
Why debian/contrib-jessie64 ? 
I'd wager that most vagrant users already have the ubuntu 14.04 box installed